networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -26,4 +26,18 @@ The Network Observability Operator is now stable and the release channel is upgr
 === Bug fix
 
 * Previously, unless the Loki `authToken` configuration was set to `FORWARD` mode, authentication was no longer enforced, allowing any user who could connect to the {product-title} console in an {product-title} cluster to retrieve flows without authentication.
-Now, regardless of the Loki `authToken` mode, only cluster administrators can retrieve flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2169468[*BZ#2169468*])
\ No newline at end of file
+Now, regardless of the Loki `authToken` mode, only cluster administrators can retrieve flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2169468[*BZ#2169468*])
+
+[id="network-observability-operator-release-notes-1-2"]
+== Network Observability Operator 1.2.0
+
+The following advisory is available for the Network Observability Operator 1.1.0:
+
+* <errata link here>
+
+[id="network-observability-operator-preparing-to-update"]
+=== Preparing for the next update 
+
+The subscription of an installed Operator specifies an update channel that tracks and receives updates for the Operator. Until the 1.2 release of the Network Observability Operator, the only channel available was `v1.0.x`. The 1.2 release of the Network Observability Operator introduces the `stable` update channel for tracking and receiving updates. You must switch your channel from the default `v1.0.x` to `stable` to recieve future Operator updates. The `v1.0.x` channel is planned to be deprecated in following releases.
```

Comment:
> You must switch your channel from the default `v1.0.x` to `stable`

`v1.0.x` isn't the default anymore... I think I would just remove the word "default" here, no?

> The `v1.0.x` channel is planned to be deprecated

I would say **is** deprecated and is planned to be removed (in the meaning deprecated=people should stop using it, even if it's still there)

Reply:
Ok I see. I guess I was thinking that it _was_ the default. Which it was, but that isn't relevant anymore so I agree it can be removed. 

Addressed deprecated wording.